src/amount.rs

```diff
@@ -0,0 +1,118 @@
+use eel::interfaces::ExchangeRate;
+use std::time::SystemTime;
+
+pub struct FiatValue {
+    pub minor_units: u64,
+    pub currency_code: String,
+    pub updated_at: SystemTime,
```

Comment:
The name `updated_at` might lead to the assumption that an `Amount` could have its fiat value updated as time goes by. I would prefer:
```suggestion
    pub converted_at: SystemTime,
```